codefori/vscode-ibmi
# Put debugged program name in IBM i batch debug session

### Changes
Resolves https://github.com/codefori/vscode-ibmi/issues/2033

This PR changes the name of the IBM i bact debug sessions to display the name of the program being debugged.
![image](https://github.com/codefori/vscode-ibmi/assets/11096890/2d98ac9f-253b-4337-9f59-c2a8b39d616d)


### How to test this PR
1. Start a batch debug session on a program and leave it open
2. Start another batch debug session on another program
3. Go to the Debug Console view
4. Check that each session has the program name in it

### Checklist
